openedx/core/djangoapps/ace_common/utils.py

```diff
@@ -0,0 +1,16 @@
+"""
+Utility functions for edx ace.
+"""
+
+
+def setup_firebase_app(firebase_credentials, app_name='fcm-app'):
+    """
+    Returns a Firebase app instance if the Firebase credentials are provided.
+    """
+    try:
+        import firebase_admin  # pylint: disable=import-outside-toplevel
+    except ImportError:
```

Comment:
Maybe log here with text: "Could not import firebase_admin package"?

Reply:
Logs added.